go/libkb/chain_link.go

```diff
@@ -1155,7 +1155,7 @@ func (c *ChainLink) Store(g *GlobalContext) (didStore bool, err error) {
 	key := DbKey{Typ: DBLink, Key: c.id.String()}
 
 	// Don't write with any aliases
-	if err = g.LocalDb.Put(key, []DbKey{}, packed); err != nil {
+	if err = g.LocalDb.Put(key, nil, packed); err != nil {
```

Comment:
Is this an optimization you tried that didn't have an effect?

Reply:
No, just a change.  I can remove it if you prefer.

Reply:
no preference